apps/openmw/mwworld/worldimp.cpp

```diff
@@ -2297,7 +2297,7 @@ namespace MWWorld
         applyLoopingParticles(player);
     }
 
-    int World::canRest ()
+    World::RestPermitted World::canRest ()
```

Comment:
I haven't ever seen enum-returning things in openmw be implemented this way. Why can't this function return type (and the enum type itself) be an integer?

Reply:
Hey, I know what enum is. :<

Had a bit of a brainfart this night concerning enum types, sorry. It's just that something looked weird in this.